### PR TITLE
fix: exception occurred in retry method

### LINKED ISSUE #611: Mocha Tests: should create a knowledge base failed
This test failed!

To configure my behavior, see [the Build Cop Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop).

If I'm commenting on this issue too often, add the `buildcop: quiet` label and
I will stop commenting.

---

commit: 4caa392e21a94493163cf5fcc0ca10447532aa75
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/fe6893f7-f5f9-4445-a123-79063ec3ca62), [Sponge](http://sponge2/fe6893f7-f5f9-4445-a123-79063ec3ca62)
status: failed
<details><summary>Test output</summary><br><pre>Command failed: node detect.v2beta1.js createKnowledgeBase -k d94da7f6-TestKnowledgeBase
detect.v2beta1.js createKnowledgeBase

Creates a new knowledge base

Options:
  --version                    Show version number                                                             [boolean]
  --documentId, -d             Full path of document in knowledge base                                          [string]
  --documentName, -m           Name of Document to Create                                  [string] [default: "testDoc"]
  --documentPath, -z           uri of document to be added                                                      [string]
  --knowledgeBaseName, -k      The name of the knowledge base to search from     [string] [default: "TestKnowledgeBase"]
  --knowledgeBaseFullName, -n  full path knowledge base                                                         [string]
  --knowledgeTypes, -t         The Knowledge type of the Document.                             [string] [default: "FAQ"]
  --languageCode, -l           The language code of the query. Defaults to "en-US".          [string] [default: "en-US"]
  --mimeType, -y               The mime_type of the Document                             [string] [default: "text/html"]
  --outputFile, -f                                                          [string] [default: "./resources/output.wav"]
  --projectId, -p              The Project ID to use. Defaults to the value of the GCLOUD_PROJECT or
                               GOOGLE_CLOUD_PROJECT environment variables.           [string] [default: "long-door-651"]
  --query, -q                  An array of text queries         [array] [required] [default: "Where is my data stored?"]
  --sessionId, -s              The identifier of the detect session. Defaults to a random UUID.
                                                              [string] [default: "99e589f0-915b-11ea-bc3e-87b764c3bfc4"]
  --help                       Show help                                                                       [boolean]

{ Error: 13 INTERNAL: Internal error encountered.
    at Object.callErrorFromStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call.js:30:26)
    at Object.onReceiveStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/client.js:175:52)
    at Object.onReceiveStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:341:141)
    at Object.onReceiveStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:304:181)
    at Http2CallStream.outputStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call-stream.js:115:74)
    at Http2CallStream.maybeOutputStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call-stream.js:154:22)
    at Http2CallStream.endCall (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call-stream.js:140:18)
    at Http2CallStream.handleTrailers (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call-stream.js:265:14)
    at ClientHttp2Stream.emit (events.js:198:13)
    at emit (internal/http2/core.js:265:8)
  code: 13,
  details: 'Internal error encountered.',
  metadata:
   Metadata {
     internalRepr: Map { 'grpc-server-stats-bin' => [Array] },
     options: {} },
  note:
   'Exception occurred in retry method that was not classified as transient' }

Error: Command failed: node detect.v2beta1.js createKnowledgeBase -k d94da7f6-TestKnowledgeBase
detect.v2beta1.js createKnowledgeBase

Creates a new knowledge base

Options:
  --version                    Show version number                                                             [boolean]
  --documentId, -d             Full path of document in knowledge base                                          [string]
  --documentName, -m           Name of Document to Create                                  [string] [default: "testDoc"]
  --documentPath, -z           uri of document to be added                                                      [string]
  --knowledgeBaseName, -k      The name of the knowledge base to search from     [string] [default: "TestKnowledgeBase"]
  --knowledgeBaseFullName, -n  full path knowledge base                                                         [string]
  --knowledgeTypes, -t         The Knowledge type of the Document.                             [string] [default: "FAQ"]
  --languageCode, -l           The language code of the query. Defaults to "en-US".          [string] [default: "en-US"]
  --mimeType, -y               The mime_type of the Document                             [string] [default: "text/html"]
  --outputFile, -f                                                          [string] [default: "./resources/output.wav"]
  --projectId, -p              The Project ID to use. Defaults to the value of the GCLOUD_PROJECT or
                               GOOGLE_CLOUD_PROJECT environment variables.           [string] [default: "long-door-651"]
  --query, -q                  An array of text queries         [array] [required] [default: "Where is my data stored?"]
  --sessionId, -s              The identifier of the detect session. Defaults to a random UUID.
                                                              [string] [default: "99e589f0-915b-11ea-bc3e-87b764c3bfc4"]
  --help                       Show help                                                                       [boolean]

{ Error: 13 INTERNAL: Internal error encountered.
    at Object.callErrorFromStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call.js:30:26)
    at Object.onReceiveStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/client.js:175:52)
    at Object.onReceiveStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:341:141)
    at Object.onReceiveStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:304:181)
    at Http2CallStream.outputStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call-stream.js:115:74)
    at Http2CallStream.maybeOutputStatus (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call-stream.js:154:22)
    at Http2CallStream.endCall (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call-stream.js:140:18)
    at Http2CallStream.handleTrailers (/tmpfs/src/github/nodejs-dialogflow/node_modules/@grpc/grpc-js/build/src/call-stream.js:265:14)
    at emit (internal/http2/core.js:265:8)
  code: 13,
  details: 'Internal error encountered.',
  metadata:
   Metadata {
     internalRepr: Map { 'grpc-server-stats-bin' => [Array] },
     options: {} },
  note:
   'Exception occurred in retry method that was not classified as transient' }

    at checkExecSyncError (child_process.js:629:11)
    at execSync (child_process.js:666:13)
    at exec (system-test/create-knowledge-base.v2beta1.test.js:26:21)
    at Context.it (system-test/create-knowledge-base.v2beta1.test.js:33:20)</pre></details>

### PR DESCRIPTION
**Pull request by munkhuushmgl**
Fixes #611 #612   🦕

### CODE REVIEW

**Comment by munkhuushmgl** [comment]
@JustinBeckwith Not sure it is a correct way to fix these issues. Found the solution from this comment :
https://github.com/googleapis/nodejs-vision/issues/575#issuecomment-611766984
PTL

File: samples/package.json [hunk 1]
```diff
@@ -16,12 +16,13 @@
   },
   "dependencies": {
     "@google-cloud/dialogflow": "^2.0.0",
+    "@grpc/grpc-js": "^1.0.3",
```
Comment (JustinBeckwith) [inline samples/package.json:19]: Why do we need that?

Comment (munkhuushmgl) [reply samples/package.json:19]: Looks like I just needed to update `google-gax` so that the grpc-js dependency to ~1.0.3

**Comment by DigitalNomad29** [comment]
Not sure if this is relevant but I just wanted to drop this here just in case. I received the error @munkhuushmgl mentioned using @google-cloud/dialogflow: 3.3.0. Below is the error log and subsequent fix:

`Error: 3 INVALID_ARGUMENT: Input text not set.
    at Object.callErrorFromStatus (C:\node_modules\@grpc\grpc-js\build\src\call.js:31:26)      
    at Object.onReceiveStatus (C:\node_modules\@grpc\grpc-js\build\src\client.js:176:52)       
    at Object.onReceiveStatus (C:\node_modules\@grpc\grpc-js\build\src\client-interceptors.js:342:141)
    at Object.onReceiveStatus (C:\node_modules\@grpc\grpc-js\build\src\client-interceptors.js:305:181)
    at C:\node_modules\@grpc\grpc-js\build\src\call-stream.js:124:78
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  code: 3,
  details: 'Input text not set.',
  metadata: Metadata {
    internalRepr: Map { 'grpc-server-stats-bin' => [Array] },
    options: {}
  },
  note: 'Exception occurred in retry method that was not classified as transient'
}
Unhandled rejection
TypeError: Cannot read property 'queryResult' of undefined
    at helloHttp (C:\index.js:107:22)
Error: Process exited with code 16
    at process.<anonymous> (C:\node_modules\@google-cloud\functions-framework\build\src\invoker.js:275:22)
    at process.emit (events.js:310:20)
    at process.EventEmitter.emit (domain.js:482:12)
    at process.exit (internal/process/per_thread.js:166:15)
    at Object.sendCrashResponse (C:\node_modules\@google-cloud\functions-framework\build\src\logger.js:37:9)
    at process.<anonymous> (C:\node_modules\@google-cloud\functions-framework\build\src\invoker.js:271:22)
    at process.emit (events.js:310:20)
    at process.EventEmitter.emit (domain.js:482:12)
    at processPromiseRejections (internal/process/promises.js:209:33)
    at processTicksAndRejections (internal/process/task_queues.js:98:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 16
npm ERR! googledialogflowclare@1.0.0 start: `npx functions-framework --target=helloHttp [--signature-type=http]`
npm ERR! Exit status 16
npm ERR!
npm ERR! Failed at the googledialogflowclare@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\AppData\Roaming\npm-cache\_logs\2020-12-10T00_25_19_724Z-debug.log

C:>npm ls @grpc/grpc-js
googledialogflowclare@1.0.0 C:
`-- @google-cloud/dialogflow@3.3.0
  `-- google-gax@2.9.2
    `-- @grpc/grpc-js@1.1.8`

**Pull request merged**